src/models/role/id.yaml

```
@@ -3,6 +3,10 @@ required:
   - id
 properties:
   id:
-    description: "Unique identifier for the actor or role."
+    description: |
+      The unique identifier for the role. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)